_includes/guide/bcda_enhancement_credentials.html

```diff
@@ -15,7 +15,7 @@
     {% include copy_snippet.md code=client_secret aco='Small REACH ACO (100 synthetic beneficiaries)' type='Client Secret' %}
```

Comment:
@phamjennifer could you please update the aria-labels for each of the 4 "copy" buttons like below? These labels are important for screen-readers 
```suggestion
    {% include copy_snippet.md code=client_secret aco='Small REACH ACO (110 synthetic beneficiaries)' type='Client Secret' %}
```

Reply:
added a commit and pushed it to this pr branch!